src/layouts/index.js

```diff
@@ -1,11 +1,11 @@
 import React from "react"
-import Header from "../components/Header/header"
-import Footer from "../components/Footer/footer"
+import Header from "../molecules/Header"
```

Comment:
Header and footer will work better as organisms, because they're gonna be containers for molecules.

Reply:
Done.